nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ConvertJSONToSQL.java

```diff
@@ -508,7 +509,12 @@ private String generateInsert(final JsonNode rootNode, final Map<String, String>
      *
      */
     protected static String createSqlStringValue(final JsonNode fieldNode, final Integer colSize, final int sqlType) {
-        String fieldValue = fieldNode.asText();
+        String fieldValue;
```

Comment:
Can you add unit test(s) to TestConvertJSONToSQL to cover this improvement? The sample data in the Jira would make an excellent unit test :)